src/controllers/cron/update_local_smartapi.js

```diff
@@ -153,7 +153,19 @@ const getOpsFromPredicatesEndpoints = async (specs) => {
 
 const updateSmartAPISpecs = async () => {
     const SMARTAPI_URL = 'https://smart-api.info/api/query?q=tags.name:translator&size=1000&sort=_seq_no&raw=1&fields=paths,servers,tags,components.x-bte*,info,_meta';
-    const res = await axios.get(SMARTAPI_URL, { headers: { 'User-Agent': userAgent } });
+    const overridesPath = path.resolve(__dirname, "../config/smartapi_overrides.json");
+    let overrides;
+    try {
+        overrides = JSON.parse((await readFile(overridesPath)));
+    } catch (error) {
+        debug(`ERROR getting API Overrides file because ${error}`);
+        return;
+    }
+    const res = await axios.get(SMARTAPI_URL, { headers: { 'User-Agent': userAgent } }).catch((err) => {
+        if(!(process.env.API_OVERRIDE === "true" && overrides)) {
```

Comment:
`res` should have the structure used by the rest of this function (and `getAPIOverrides`) re-created if the request fails, so that overrides can be created properly.

Reply:
Hm, I think I misunderstood you about this. Do you mean that if `API_OVERRIDE` and `overrides` exist, the error is logged, and overrides are created with the local file instead? And, if `API_OVERRIDE` and `overrides` don't exist, the error is thrown?

Reply:
If `API_OVERRIDE` is `true` and the [config `only_overrides`](https://github.com/biothings/BioThings_Explorer_TRAPI/blob/6b6d07106b8c04aeca3723a1d5c9a7dce0b9fd74/src/config/smartapi_overrides.json#L3) is also `true`, when the initial smartAPI request fails, we should see an error saying it failed, and it should proceed to create the smartAPI specs from only what's provided by the overrides config.

If both of those are not true, then the error should be logged to explain why updating smartAPI specs failed entirely.